QGL/BasicSequences/Decoupling.py

```diff
@@ -51,18 +52,19 @@ def CPMG(qubit, numPulses, pulseSpacing, calRepeats=2, showPlot=False):
 	-------
 	plotHandle : handle to plot window to prevent destruction
 	"""
-	#First setup the t-180-t block
-	CPMGBlock = [Id(qubit, (pulseSpacing-qubit.pulseParams['length'])/2),
-								 Y(qubit), Id(qubit, (pulseSpacing-qubit.pulseParams['length'])/2)]
-
-	seqs = [[X90(qubit)] + CPMGBlock*rep + [X90(qubit), MEAS(qubit)] for rep in numPulses]
+    #First setup the t-180-t block
```

Comment:
Is there a setting to standardize putting a space between the `#` and the text of the comment?